### PR TITLE
Fix simple module, update readme

### PR DESCRIPTION
**Pull request by cheukt**
updated client as well
as part of testing, found a small issue with our validation code so fixed that as well

also made all the logs info so it'll show up even if not in debug mode

@dannenberg think this is what you wanted

### CODE REVIEW

File: examples/simple_module/client.py [hunk 1]
```diff
@@ -7,9 +7,9 @@
 
 
 async def connect():
-    creds = Credentials(type="<your authentication type here>", payload="<your authentication payload here>")
-    opts = RobotClient.Options(refresh_interval=0, dial_options=DialOptions(credentials=creds), log_level=logging.DEBUG)
-    return await RobotClient.at_address("<your robot uri here>", opts)
+    # creds = Credentials(type="<your authentication type here>", payload="<your authentication payload here>")
```
Comment (maximpertsov) [inline examples/simple_module/client.py:10]: assuming you meant to uncomment

```suggestion
    creds = Credentials(type="<your authentication type here>", payload="<your authentication payload here>")
```

Comment (maximpertsov) [reply examples/simple_module/client.py:10]: nvm i see we are just changing to insecure

Comment (purplenicole730) [reply examples/simple_module/client.py:10]: Might have to un-import `Credentials` now that we're not using it anymore.

**Review by purplenicole730 — APPROVED** [review]
LGTM

File: examples/simple_module/client.py (hunk 1 above)
Comment (cheukt) [reply examples/simple_module/client.py:10]: ah that's unintentional, will revert that part

**Review by dannenberg — APPROVED** [review]
Thank you!! <3

**Pull request merged**